Common/Systems/Questing/Quests/MainPath/EoWQuest.cs

```diff
@@ -27,6 +27,7 @@ public override List<QuestStep> SetSteps()
 	{
 		return
 		[
+			new InteractWithNPC(ModContent.NPCType<MorvenNPC>(), this.GetLocalization("BlacksmithInput"), this.GetLocalization("MorvenDialogue")),
 			new ConditionCheck((_) => SubworldSystem.Current is RavencrestSubworld && NPC.AnyNPCs(ModContent.NPCType<MorvenNPC>()), 1, this.GetLocalization("MorvenRaven")),
```

Comment:
Adding a new step will shift people back by a step on their saved progression. Again suggests that we have StringIDs for every quest step, using them in IO.

Reply:
For quest steps, it would only mean altering save/load code to refer to steps by named IDs rather than by their index, which changes is the diff here.